docs_src/data_block.ipynb

```diff
@@ -1023,6 +1023,356 @@
     "  - [`CollabList`](/collab.html#CollabList) for collaborative filtering"
```

Comment:
<p>Again, `__getitem__` is standard python.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
I see. 
By the way, I remembered you have figured out a way to show changes in jupyter notebook nicely when working with git. But I don't remember where to find it, could you paste me a link to it? Thanks